### ISSUE TITLE
Citation

### ISSUE THREAD

**Issue by vieirasaulo** [Deltares/pandamesh#11]
Hi there,

How can I cite pandamesh?

Best wishes,

Saulo

**Comment by Huite** [comment]
Hi @vieirasaulo,

Good question! I have a list of Python packages I should submit and entry for in Journal of Open Source Software...

Until then, I think citing like the [Dask docs](https://dask-local.readthedocs.io/en/latest/cite.html#citations) suggest will do:

> Dask Development Team (2016). Dask: Library for dynamic task scheduling
> URL http://dask.pydata.org
>
> Bibtex:
> ```
> @Manual{,
>   title = {Dask: Library for dynamic task scheduling},
>   author = {{Dask Development Team}},
>   year = {2016},
>   url = {http://dask.pydata.org},
> }
> ```


Along those lines for pandamesh:

Pandamesh Development Team (2023). Pandamesh: Library for mesh generation from geospatial vector data
URL https://github.com/Deltares/pandamesh

Bibtex:
@Manual{,
  title = { Pandamesh: Library for mesh generation from geospatial vector data},
  author = {{Pandamesh Development Team}},
  year = {2023},
  url = {https://github.com/Deltares/pandamesh},
}

**Comment by vieirasaulo** [comment]
Hey, thanks for getting back to me. If I may, I've been citing other packages as @software using Zenodo or PyPI, and was wondering if it would be cool with you to cite it like this:

@software{Deltares2021,
   author = {Pandamesh Development Team}},
   title = {Pandamesh: Library for mesh generation from geospatial vector data},
   url = {https://pypi.org/project/pandamesh/},
   year = {2023},
   note = {Computer Software, \textit{Python Software Foundation}},
   publisher = {Python Software Foundation},
   organization = {Deltares},
   version = {0.1.2},
}

Does that make sense?

**Comment by Huite** [comment]
Yes, fine by me. I'm just not sure whether the Python Software Foundation is the appropriate publisher. This is true for Python (or specifically CPython), but not for all packages. I think pandamesh is "published" on PyPI, conda, and github.

We could also interpret github as the main published site, and so you could list github as the publisher. That also what one of these answer on stackexchange suggests:
https://academia.stackexchange.com/questions/14010/how-do-you-cite-a-github-repository

**Comment by vieirasaulo** [comment]
Alright! ir makes sense.  Thanks for the answer!
